src/selenium_screenshot.py

```diff
@@ -112,7 +113,13 @@ def take_selenium_screenshot(url, file_name, width=1000, height=500, **kwargs):
     WebDriverWait(browser, timeout).until(
         EC.presence_of_element_located((By.XPATH, app_selector_xpath))
     )
-    time.sleep(timeout)
+
```

Comment:
You should add a `time.sleep(timeout)` here, for the initial wait until that the browser loads the right data.